glusterd2/commands/volumes/volume-create.go

```diff
@@ -73,6 +73,25 @@ func createVolinfo(req *api.VolCreateReq) (*volume.Volinfo, error) {
 		v.Type = volume.DistReplicate
 	}
 
+	if req.DisperseData != 0 || req.DisperseRedundancy != 0 {
```

Comment:
Quoting Xavi from [EC xlator change](https://review.gluster.org/#/c/19028/):

> In current implementation of glusterd, it's allowed to not pass the 'redundancy' argument on a 'volume create' command. In this case, the redundancy is automatically computed to try to get an optimal configuration given the 'disperse' value.

The above functionality can be added in glusterd2 as well.